src/esn.calendar.libs/app/components/event/form/modals/edit-instance-or-series-modal.less

```diff
@@ -3,4 +3,12 @@
     font-size: 1.1em;
     font-weight: 400;
   }
+  .suggest-form {
+    .modal-dialog {
+      position: relative !important;
```

Comment:
Why do we need `important` here?
If it is mandatory, it deserves a comment to explain why? To override what?

Reply:
Done!

Reply:
Done!